### PR TITLE
GH-371 Fix retrieval_cache_period_seconds to be set to 0

### PR DESCRIPTION
**Pull request by maheshjfrog**
Fixes: #371

### CODE REVIEW

**Review by chb0github — APPROVED** [review]

**Review by chb0github — APPROVED** [review]
Is there anyway we can do this in 1 commit when the change is made?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.25.1 (Mar 28, 2022)
+
+BUG FIXES:
+
+* Fix retrieval_cache_period_seconds to be set to 0 for artifactory_*_repository resources. [GH-373]
```
Comment (alexhung) [inline CHANGELOG.md:5]: The change affects only remote repos, right? Let's be more specific in the changelog.

**Review by alexhung — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (maheshjfrog) [reply CHANGELOG.md:5]: i see "retrieval_cache_period_seconds" ... is available for both remote and virtual repos.

**Comment by alexhung** [comment]
> Is there anyway we can do this in 1 commit when the change is made?

@chb0github Yes, if we use the 'Create as draft PR' feature 😄. So create a draft PR, use the PR number to update changelog, commit/push, make PR public.

File: CHANGELOG.md (hunk 1 above)
Comment (alexhung) [reply CHANGELOG.md:5]: The change affects `RemoteRepositoryBaseParams` only.

Comment (maheshjfrog) [reply CHANGELOG.md:5]: will update the change log to be specific.

**Review by chb0github — APPROVED** [review]

**Review by alexhung — APPROVED** [review]

**Pull request merged**